adapters/AppLovin/AppLovinAdapter/RTB/GADMAppLovinRtbBannerRenderer.m

```diff
@@ -0,0 +1,136 @@
+//
+//  GADMAppLovinRtbBannerRenderer.m
+//  Adapter
+//
+//  Created by Christopher Cong on 7/17/18.
+//  Copyright © 2018 Google. All rights reserved.
+//
+
+#import "GADMAppLovinRtbBannerRenderer.h"
+#import "GADMAdapterAppLovinUtils.h"
+#import "GADMAdapterAppLovinConstant.h"
+
+#import <AppLovinSDK/AppLovinSDK.h>
+
+@interface GADMAppLovinRtbBannerRenderer () <GADMediationBannerAd, ALAdLoadDelegate, ALAdDisplayDelegate, ALAdViewEventDelegate>
+
+/// Data used to render an RTB banner ad.
+@property (nonatomic, strong) GADMediationBannerAdConfiguration *adConfiguration;
+
+/// Callback object to notify the Google Mobile Ads SDK if ad rendering succeeded or failed.
+@property (nonatomic, copy) GADBannerRenderCompletionHandler renderCompletionHandler;
+
+/// Delegate to notify the Google Mobile Ads SDK of banner presentation events.
+@property (nonatomic, strong, nullable) id<GADMediationBannerAdEventDelegate> adEventDelegate;
+
+/// Controlled Properties
+@property (nonatomic, strong) ALSdk *sdk;
+@property (nonatomic, assign) ALAdSize *adSize;
+@property (nonatomic, strong) ALAdView *adView;
+
+@end
+
+@implementation GADMAppLovinRtbBannerRenderer
+
+- (instancetype)initWithAdConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration
+                      completionHandler:(nonnull GADBannerRenderCompletionHandler)handler {
+    self = [super init];
+    if (self) {
+        self.adConfiguration = adConfiguration;
+        self.renderCompletionHandler = handler;
+        
+        // Convert requested size to AppLovin Ad Size.
+        self.adSize = [GADMAdapterAppLovinUtils adSizeFromRequestedSize:adConfiguration.adSize];
+        self.sdk = [GADMAdapterAppLovinUtils retrieveSDKFromCredentials:adConfiguration.credentials.settings];
```

Comment:
Are you sure this is the correct `serverParameters` - quite different than how it's done on Android.